### PR TITLE
add directionalHintFixed as a prop in hover card

### PR DESCRIPTION
**Pull request by chaofeng2014**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Make hover card choose position smartly

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.tsx [hunk 1]
```diff
@@ -68,6 +68,7 @@ export class ExpandingCard extends BaseComponent<IExpandingCardProps, IExpanding
       theme,
       styles: customStyles,
       compactCardHeight,
+      directionalHintFixed = true,
```
Comment (atneik) [inline packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.tsx]: Put the defaults in `public static defaultProps`

Should we also not expose this prop in HoverCard?

File: packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.Props.ts [hunk 2]
```diff
@@ -89,6 +89,11 @@ export interface IExpandingCardProps extends React.HTMLAttributes<HTMLDivElement
   styles?: IExpandingCardStyles;
 
   /**
+   * Make callout content fixed on the set side
+   */
```
Comment (atneik) [inline packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.Props.ts]: mention `@default true` in comments

**Comment by betrue-final-final** [comment]
Hey Chao, what does this do exactly? I'm getting a visual change.

**Comment by chaofeng2014** [comment]
@betrue-final-final the visual is not cause by this PR I think, I just make the hover card smart enough to choose where to show itself

**Pull request merged**